bundles/org.openhab.binding.miio/src/main/resources/database/huayi.light.fanwy2-miot.json

```diff
@@ -192,7 +196,7 @@
 				"piid": 3,
 				"friendlyName": "Presets - Pre Brightness",
 				"channel": "pre-brightness",
-				"type": "Number:Dimensionless",
+				"type": "Dimmer",
 				"unit": "percentage",
```

Comment:
Are units still used if you use a Dimmer instead?

Reply:
In my testing it does not seem to harm. 
But... I did notice when I use in the item formatting %unit% it does not accept that for a dimmer, 
so there is no value in having the unit specified. 
Hence I removed the units from the dimmers now.